static/js/errorlogs/viewlogs.js

```diff
@@ -1,4 +1,20 @@
 MEDUSA.errorlogs.viewlogs = function() {
+    if (MEDUSA.config.fanartBackground) {
```

Comment:
Please don't do this.... if you need to resuse the function then add it to the common init file. 😞